lib/rspec.rb

```diff
@@ -9,7 +9,7 @@ def setup_rspec
   # folder for rspec
   run 'mkdir spec/codebase'
   run 'mkdir spec/fabricators'
-  run 'mkdir spec/features'
+  run 'mkdir spec/systems'
```

Comment:
Is it `systems` or `system` in the end? I thought we discuss about it during Nimbl3 Growth and you said it was `system` 😜

Reply:
From Rspec, they said that it is `system` because they (we also) use `config.infer_spec_type_from_file_location!` in `rails_helpers` it mean that every file under `system` folder it will run with `system type`

But for us, I think for the convention of the template, we have `models, controllers, ...` I think it will good if we go with `systems`

And if we go with `systems`, we have to put `type: :system` on each file like this

```
describe 'Sign in', type: :system
  ...
end
```

How do you think?

Reply:
@anduonghien @olivierobert if we still differentiate the specs with `type: :system`, why not have the same old `features` directory. For me its more descriptive than `system(s)`

Reply:
Let's settle on `system` as 1) it's the default name and 2) it remove the need to add `type: :system` to all tests (time saving FTW)

Reply:
Ok @olivierobert, will change it